libs/design/src/scss/theming/illuminate/illuminate.spec.scss

```diff
@@ -3,12 +3,8 @@
 @import './illuminate.scss';
 
 @include describe('illuminate') {
```

Comment:
Should be a skip, not a change of braces.

Reply:
Would this be done by disabling the sass-lint or inserting something inside the function to skip?

Reply:
Both. For this PR, we need to just disable.